### PR TITLE
only call event handlers if they are functions

### PR DESCRIPTION
**Pull request by tgabi333**
We found a crach setting event handlers to undefined

### CODE REVIEW

**Comment by fippo** [comment]
whoops, not sure how I missed that... can you rebase please?

File: rtcpeerconnection.js [hunk 1]
```diff
@@ -448,7 +448,7 @@ module.exports = function(window, edgeVersion) {
       // polyfill since RTCIceGatherer.state is not implemented in
       // Edge 10547 yet.
       iceGatherer.state = end ? 'completed' : 'gathering';
-      if (self.transceivers.iceCandidates !== null) {
+      if (typeof self.transceivers.iceCandidates === 'function') {
```
Comment (fippo) [inline rtcpeerconnection.js]: oh... this one was wrong and is probably the source of the conflict.

**Comment by tgabi333** [comment]
@fippo: rebased

**Comment by tgabi333** [comment]
ping @fippo

**Comment by fippo** [comment]
E_TOO_MANY_NOTIFICATIONS. Thank you!

**Comment by fippo** [comment]
published as 1.1.5

**Comment by tgabi333** [comment]
thanks :)

**Pull request merged**